server/src/main/java/org/opensearch/index/codec/PerFieldMappingPostingFormatCodec.java

```diff
@@ -67,6 +75,12 @@ public PerFieldMappingPostingFormatCodec(Mode compressionMode, MapperService map
         super(compressionMode);
         this.mapperService = mapperService;
         this.logger = logger;
+        fuzzySetFactory = new FuzzySetFactory(
+            Map.of(
+                IdFieldMapper.NAME,
+                new FuzzySetParameters(() -> mapperService.getIndexSettings().getDocIdFuzzySetFalsePositiveProbability())
+            )
```

Comment:
Does the compression mode not apply on fuzzy set?

Reply:
Do you mean lz4/deflate/zstd, etc?
They apply to sroted fields.

Also, the delegate is using the default postings format which should apply default behaviour of L90PostingsFormat here.